crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -202,13 +202,37 @@ type EndOfEpochTransaction {
 }
 
 type Epoch {
+	"""
+	The epoch's id
```

Comment:
Explain that this is a sequence number? (i.e. the first epoch is epoch 0, and it increases one at a time).